content/en/welcome/_index.md

```diff
@@ -72,5 +72,6 @@ There are many ways you can stay involved with the Kubernetes Contributor commun
 
 {{% /blocks/section %}}
 
-
+# Mastodon Account Verification Code
+<a rel="me" href="https://hachyderm.io/@K8sContributors"></a>
```

Comment:
This is the wrong page for verification
It should go here:
https://github.com/kubernetes/contributor-site/blob/master/content/en/_index.md

It also does not need a header, the link just has to be present on the site for it to detect

Reply:
I apparently forgot how to Markdown.